DeepSpeech/Dockerfile.train

```diff
@@ -144,17 +148,26 @@ RUN git checkout $ds_branch
 WORKDIR $DS_DIR
 
 RUN pip install --upgrade pip==20.0.2 wheel==0.34.2 setuptools==46.1.3
-RUN DS_NOTENSORFLOW=y pip install --upgrade --force-reinstall -e .
+
+# Build CTC decoder first, to avoid clashes on incompatible versions upgrades
+RUN cd native_client/ctcdecode && make NUM_PROCESSES=$(nproc) bindings
+RUN pip install --upgrade native_client/ctcdecode/dist/*.whl
+
+# Install STT
+# No need for the decoder since we did it earlier
+# TensorFlow GPU should already be installed on the base image,
+# and we don't want to break that
+RUN DS_NODECODER=y DS_NOTENSORFLOW=y pip install --upgrade --force-reinstall -e .
+
 RUN pip install --upgrade tensorflow-gpu==1.15.4
 
-RUN TASKCLUSTER_SCHEME="https://community-tc.services.mozilla.com/api/index/v1/task/project.deepspeech.tensorflow.pip.%(branch_name)s.%(arch_string)s/artifacts/public/%(artifact_name)s" python util/taskcluster.py \
-	--target="$(pwd)" \
-	--artifact="convert_graphdef_memmapped_format" \
-	--branch="r1.15" && chmod +x convert_graphdef_memmapped_format
+# Tool to convert output graph for inference
```

Comment:
since stt removed support for tensorflow protobuf, we probably dont need that anymore